static/app/components/arithmeticBuilder/token/function.tsx

```diff
@@ -745,6 +746,33 @@ const FunctionWrapper = styled('div')<{state: 'invalid' | 'warning' | 'valid'}>`
   }
 `;
 
+const ArgumentsGridWrapper = styled('div')`
+  display: flex;
+  align-items: center;
+  position: relative;
+  height: 100%;
+  flex-shrink: 1;
+  flex-grow: 0;
+`;
+
+const ArgumentGridCell = styled('div')`
+  display: flex;
+  align-items: center;
+  position: relative;
+  height: 100%;
+  flex: 0 1 auto;
+  max-width: fit-content;
+
+  > div:first-child input {
+    width: 100% !important;
+    max-width: fit-content !important;
+    min-width: 0 !important;
+    overflow: hidden !important;
```

Comment:
need to specifically target the input here otherwise `overflow: hidden` makes the combo box also disappear